Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Card.css';
+
+function CardContent() {
+  return (
+    <div className="card-content">
+      <h3>Get Started with React</h3>
```

Comment:
good job with remembering to use good semantic HTML with the h3 and p tags. It's easy to get lazy in React with your HTML so I'm glad you are avoiding that mistake and doing things right.